libraries/standard/mqtt/lexicon.txt

```diff
@@ -11,6 +11,7 @@ config
 connack
 connectserializeroverride
 const
+coverity
```

Comment:
What is the purpose of this?

Reply:
The lexicon file is for spellchecking on CI; it whitelists certain words that are not in the spellchecker's dictionary so that the test will pass.

Reply:
Got it, thank you.